--- Compress-LZO-1.08/Makefile.PL.orig	2005-07-13 16:51:48.000000000 -0300
+++ Compress-LZO-1.08/Makefile.PL	2005-07-13 16:55:28.000000000 -0300
@@ -5,7 +5,7 @@
 # Change the path on the next line to point to the directory where
 # <lzoconf.h> and <lzo1x.h> can be found.
 
-#$LZO_INCLUDE = '-I/usr/include/lzo';
+$LZO_INCLUDE = '-I/usr/include/lzo';
 #$LZO_INCLUDE = '-I/usr/local/include';
 #$LZO_INCLUDE = '-I/usr/local/include/lzo';
 
@@ -21,7 +21,7 @@
 WriteMakefile(
                 NAME   	 	=> 'Compress::LZO',
 		VERSION_FROM  	=> 'LZO.pm',
-		LIBS	 	=> [ "$LZO_LIB -llzo" ],
+		LIBS	 	=> [ "$LZO_LIB -llzo2" ],
 	      	INC      	=> "$LZO_INCLUDE",
 		'dist'	 	=> {COMPRESS=>'gzip', SUFFIX=>'gz'},
 	     );
